### ISSUE TITLE
Unpublished from NPM?

### ISSUE THREAD

**Issue by BenjaminWFox-Lumedic** [maerzhase/webpack-clear-require-cache-plugin#3]
Looks like the package has been removed. Is it planned to come back?

**Comment by maerzhase** [comment]
Yes it will!

An account that previously published one version of this packages got deleted. I don't understand how they could delete the package from the registry. When i am trying to publish now I receive:

```
npm ERR! code E403
npm ERR! 403 403 Forbidden - PUT https://registry.npmjs.org/webpack-clear-require-cache-plugin - webpack-clear-require-cache-plugin cannot be republished until 24 hours have passed.
npm ERR! 403 In most cases, you or one of your dependencies are requesting
npm ERR! 403 a package version that is forbidden by your security policy.
```

So I guess I have to wait.

**Comment by maerzhase** [comment]
I've just republished the package in version `0.0.5` unfortunatly there was a mistake done by npm and all old version are not available anymore, but installing `0.0.5` works again. I just tested it.

Sorry for the inconveniences, folks!
